### ISSUE TITLE
Bungee chat does not work on multiple Bungeecord with RedisBungee

### ISSUE THREAD

**Issue by nhanledev** [Zrips/CMI#7104]
### Description of Issue

If two players are on different servers, but connected via the same bungeecord, they can see public chat message at network level.
If the same players are on different servers, connected via different bungeecord, thepublic chat message only show ones from current server.

The same thing happens with `/msg` command.

Is this behavior intended to work like that? Is multiple Bungeecord with RedisBungee support?

### Version Information

```shell
[07:49:11 INFO]: --------------------------------------------------
[07:49:11 INFO]: CMI: 9.2.3.0 BungeeCord CMIB  MySQL
[07:49:11 INFO]: CMILib: 1.2.3.1 
[07:49:11 INFO]: Server: Paper(81) 1.19-R0.1-SNAPSHOT
[07:49:11 INFO]: CMI economy: Enabled Vault: 1.7.3-CMI CMI Chat: Enabled 
[07:49:11 INFO]: Modules -> 19 enabled 37 disabled: spawnerProximity, cuffed, mirror, tablist, versionCheck, moneyCheque, skin, noTarget, elytraBoost, cmiPlaytime, elytraLaunch, bossBarCompass, coloredArmor, playerCombat, headDrop, attachedCommands, spawnerCharge, interactiveCommands, namePlates, timedCommands, homeInteractions, hpBossBar, launchPad, armorstand, portals, anvilRenameColor, jail, spawners, paintingEditor, dynamicSigns, durabilityLoss, flightCharge, disabledEnchants, firstJoinMessages, silentChest, warmup, totemBehavior
[07:49:11 INFO]: --------------------------------------------------
```

### Relevant Plugins

CMIB1.0.2.0.jar

### Agreements

- [X] My server is supported by CMI.
- [X] My version of CMI at the time of this report is up to date.
- [X] I have searched the github and asked around before making this report.

**Comment by dan28000** [comment]
redis syncs players not commands/messages the plugin needs to be supporting redis to be able to sync like messages rather create your own plugin for PM or find something with redis

considering state of proxy support which is kinda bad where you can't event send prefixes in PM over bungee

I'm admin on server where we have custom plugin with redis for usage like this

Yea it would be awesome to see mysql/bungee syncing with overhaul of bunch of system like bans warns vanish homes warps etc...

**Comment by nhanledev** [comment]
Yes I hope there would be more support for the basic stuff like tpa msg to work across multi bungee instances.

**Comment by nhanledev** [comment]
Hello Zrips do you have plan to support RedisBungee or do you have any reference for me to implement it myself?

**Comment by nhanledev** [comment]
Can you please move all the BungeeCord Forward packet to its own channel?
